docs/user_guide/transformation/ArcsinTransformer.rst

```diff
@@ -0,0 +1,79 @@
+.. _reciprocal:
```

Comment:
here, we need to replace _reciprocal by for example `_arcsin`. Important the _ before the name.